tests/ospf/conftest.py

```diff
@@ -0,0 +1,75 @@
+import pytest
+import re
+from tests.common.config_reload import config_reload
+import time
+
+
+@pytest.fixture(scope="module")
+def ospf_Bfd_setup(duthosts, rand_one_dut_hostname, nbrhosts, request):
+    if request.config.getoption("neighbor_type") != "sonic":
+        pytest.skip("Neighbor type must be sonic")
+
+    duthost = duthosts[rand_one_dut_hostname]
+    setup_info = {"nbr_addr": {}, "int_info": {}}
+
+    cmd = "show ip route bgp"
+    bgp_neighbors = duthost.shell(cmd)["stdout"]
+    for line in bgp_neighbors.splitlines():
+        match = re.search(r"(\d+\.\d+\.\d+\.\d+)", line)
+        if match:
+            nbr_ip = match.group(1)
+            setup_info["nbr_addr"][nbr_ip] = None
+
+    for nbr_name, nbr_info in nbrhosts.items():
+        nbr_ip = setup_info["nbr_addr"].get(nbr_info["ip"])
+        if nbr_ip:
+            configure_ospf_and_bfd(
+                nbr_info["host"],
+                nbrhosts[nbr_name].hostname)
+            setup_info["nbr_addr"][nbr_ip] = {"hostname": nbr_info["hostname"]}
+
+    setup_info["int_info"] = get_ospf_neighbor_interface()
```

Comment:
Hi @Ghulam-Bahoo/@asraza07  just checking if this test is passing from your end? I have trouble running from my side, some of the method/arguments are not called correctly for example this one here is missing argument `host`.

I tried to fix myself but eventually could not get the OSPF established